dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python module for making simple text/console-mode user interfaces"
+HOMEPAGE="http://pythondialog.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythondialog//${PV}/python3-${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc examples"
```

Comment:
are `doc` and `examples` totally independent of each other?  
Looking at the code written below seems to suggest there might be some relation which might make things simpler?

Like `REQUIRED_USE="examples? ( doc )"`, might make it easier to use sphinx? Just a thought.

Reply:
maybe lose the doc altogether (see https://dev.gentoo.org/~mgorny/python-guide/distutils.html#building-documentation-via-sphinx)

Reply:
> maybe lose the doc altogether (see https://dev.gentoo.org/~mgorny/python-guide/distutils.html#building-documentation-via-sphinx)

That's what @epsilon-0 suggested with `distutils_enable_sphinx`, right? Unless I'm misunderstanding your comment here.

Reply:
@ILMostro https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/distutils-r1.eclass#n284
i meant that you should definitely add the `distutils_enable_sphinx` which automaticlaly adds the `docs` keyword, and on top of it, I thought maybe if you want to install examples then maybe you also want to enable docs.